Re-add missing dependency (cryptography)

lukpueh opened the pull request:
**Fixes issue #**:
Fixes comment in unrelated PR https://github.com/in-toto/in-toto/pull/169#discussion_r162169742

**Description of the changes being introduced by the pull request**:
When `securesystemslib` was factored out into a separate package, in-toto also delegated the dependency of `cryptography` to `securesystemslib.` By adding gpg support to `in-toto` `cryptography` again becomes a dependency.

This commit adds `cryptography` to `requirements.tx`t and `cryptography>=2.1.3` (same as specified by the required `securesystemslib==0.10.8`) to `setup.py`.

**Please verify and check that the pull request fulfills the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

Commit:
Re-add missing dependency (cryptography)

When securesystemslib was factored out into a separate package,
in-toto's also delegated the dependency of cryptography to
securesystemslib.
By adding gpg support to in-toto cryptography again becomes a
dependency.
This commit adds 'cryptography' to requirements.txt and
cryptography>=2.1.3 (same as specified by the required
securesystemslib==0.10.8) to setup.py.

vladimir-v-diaz commented on setup.py:
```diff
@@ -65,8 +65,8 @@
   packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
-  install_requires=["six", "securesystemslib==0.10.8", "attrs",
-                    "python-dateutil", "iso8601"],
+  install_requires=["six", "cryptography>=2.1.3", "securesystemslib==0.10.8",
```
You explain in the pull request summary that the cryptography dependency is set to `>=2.1.3` because that's the minimum version of cryptography supported by `securesystemslib`.

Should you add a note/comment in `setup.py` to remind of that fact?    Suppose `securesystemslib` increases the minimum version of cryptography, will in-toto have to follow suit?

lukpueh replied:
I'll add a comment.

Commit:
Remove cryptography version requirement

The minimum version is dictated by securesystemslib, which specifies
cryptography>=2.1.3.

lukpueh commented:
Thanks for the review, @vladimir-v-diaz!
